app/models/work.rb

```diff
@@ -13,6 +13,8 @@ class Work < ApplicationRecord
   has_many :attached_files, dependent: :destroy
   has_many :keywords, dependent: :destroy
 
+  accepts_nested_attributes_for :attached_files, :contributors, :keywords
```

Comment:
Helped with unpermitted parameters resulting from `attached_files_attributes` submitted to the WorkForm.

Reply:
It's my understanding this isn't necessary with reform. We don't use it for other types of attached things.

Reply:
@jcoyne I'll try taking this line out.

Reply:
done.